bridges/relays/lib-substrate-relay/src/cli/relay_messages.rs

```diff
@@ -216,4 +218,18 @@ where
 		)
 		.await
 	}
+
+	/// Add relay guards if required.
+	async fn start_relay_guards(
+		target_client: &impl Client<Self::Target>,
+		enable_version_guard: bool,
+	) -> relay_substrate_client::Result<()> {
+		if enable_version_guard {
```

Comment:
I would remove the `enable_version_guard: bool` and move the `if` outside of the function.

Reply:
yes, it could be all simplified, I have TODO for that here: https://github.com/paritytech/polkadot-sdk/issues/5923,
we have three copies of this exact function